cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from datetime import timedelta
+
+from fixtures.pytest_store import store
+from cfme.optimize.bottlenecks import Bottlenecks
+from utils import conf
+from utils.appliance.implementations.ui import navigate_to
+from utils.appliance import get_or_create_current_appliance
+from utils.ssh import SSHClient
+from utils.version import current_version
+
+db_tbl = store.current_appliance.db['bottleneck_events']
+db_events = store.current_appliance.db.session.query(db_tbl.timestamp,
+    db_tbl.resource_type, db_tbl.resource_name, db_tbl.event_type, db_tbl.severity, db_tbl.message)
+
+
+@pytest.fixture(scope="session")
+def extend_db_partition():
+    app = get_or_create_current_appliance()
```

Comment:
At the same time... I don't see a reason why not to leverage Sprout for this. But like I said - `test_db_migrate` already does this and I'm betting on it working correctly. If it doesn't, I do need to know.